spec/defines/sslcertificate_spec.rb

```diff
@@ -4,25 +4,27 @@
   describe 'when managing a ssl certificate' do
     let(:title) { 'certificate-testCert' }
     let(:params) { {
-        :name       => 'testCert',
-        :password   => 'testPass',
-        :location   => 'C:\SslCertificates',
-        :thumbprint => '07E5C1AF7F5223CB975CC29B5455642F5570798B',
-        :root_store => 'LocalMachine',
-        :store_dir  => 'My',
+      :name       => 'testCert',
+      :password   => 'testPass',
+      :location   => 'C:\SslCertificates',
+      :thumbprint => '07E5C1AF7F5223CB975CC29B5455642F5570798B',
+      :root_store => 'LocalMachine',
+      :store_dir  => 'My',
     } }
 
     it { should contain_exec('Install-testCert-SSLCert').with(
       'command'  => 'c:\temp\import-testCert.ps1',
       'onlyif'   => 'c:\temp\inspect-testCert.ps1',
       'provider' => 'powershell'
-    )}
+    )
+    }
 
     it { should contain_file('import-testCert-certificate.ps1').with(
       'ensure'  => 'present',
       'path'    => 'C:\\temp\\import-testCert.ps1',
       'require' => 'File[C:\temp]'
-    )}
+    )
+    }
```

Comment:
perhaps, since this is a multi-line expression, change it to:

```ruby
it do
  should contain_file('import-testCert-certificate.ps1').with(
       'ensure'  => 'present',
       'path'    => 'C:\\temp\\import-testCert.ps1',
       'require' => 'File[C:\temp]'
   )
end
```